src/meta/src/hummock/manager/mod.rs

```diff
@@ -1270,6 +1418,61 @@ where
             .await;
         }
 
+        // Warn of table_ids that is not found in expected compaction group.
+        // It indicates:
+        // 1. Either these table_ids are never registered to any compaction group. This is FATAL
+        // since compaction filter will remove these valid states incorrectly.
+        // 2. Or the owners of these table_ids have been dropped, but their stale states are still
+        // committed. This is OK since compaction filter will remove these stale states
+        // later.
```

Comment:
How can we detect this case?

Reply:
> How can we detect this case?

it might be `committed SST sanity check`. @zwang28

Reply:
This "Warn of table_ids..." comment is from main and is no longer holds in this PR. In main we also don't (cannot) distinguish case 1 from case 2, but write logs to help troubleshooting.

Besides,
> 1. Either these table_ids are never registered to any compaction group. This is FATAL

Previously this will raise error. Now the table_id may not match in any groups (L1455), then it is not committed and ignored silently.

Reply:
> This "Warn of table_ids..." comment is from main and is no longer holds in this PR. In main we also don't (cannot) distinguish case 1 from case 2, but write logs to help troubleshooting.

thanks